deployment/helm/synopsys-alert/templates/alert-environ-configmap.yaml

```diff
@@ -1,13 +1,20 @@
 apiVersion: v1
 kind: ConfigMap
 data:
+  {{- if .Values.alert.hostname }}
+  ALERT_HOSTNAME: {{ .Values.alert.hostname | quote }}
+  {{- else }}
+  ALERT_HOSTNAME: {{ .Release.Name | quote }}-alert
+  {{- end}}
+  {{- if .Values.alert.port }}
+  ALERT_SERVER_PORT: {{ .Values.alert.port | quote }}
```

Comment:
I don't think "ALERT_SERVER_PORT" *needs* to be configurable. But this is okay if you want it :) 

What do you think @msenmurugan ?

Reply:
I have removed '-alert' from the hostname.  If it should be added back then I can add it back.

Reply:
Will that impact existing customers using the helm charts? Would changing the host name impact the volumes or any of the data?

Reply:
the original host name was always localhost and that is the default so this should not affect existing customers.  The <release name>-alert was a way to provide a different default host name unique to alert if the hostname was missing from values.yml